x-pack/plugins/case/server/routes/api/cases/configure/post_configure.ts

```diff
@@ -59,7 +64,12 @@ export function initPostCaseConfigure({ caseConfigureService, caseService, route
         });
 
         return response.ok({
-          body: CaseConfigureResponseRt.encode({ ...post.attributes, version: post.version ?? '' }),
+          body: CaseConfigureResponseRt.encode({
+            ...post.attributes,
+            // Reserve for future implementations
+            connector: { ...post.attributes.connector, fields: null },
```

Comment:
I feel like we also need to `transformESConnectorToCaseConnector` even if the `fields` attribute is null for consistency.